drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -13,6 +13,10 @@ class HmisSchema::Enrollment < Types::BaseObject
     field :project, Types::HmisSchema::Project, null: false
     field :entry_date, GraphQL::Types::ISO8601DateTime, null: true
     field :exit_date, GraphQL::Types::ISO8601DateTime, null: true
+    field :assessments, HmisSchema::Assessment.page_type, null: false
+    field :events, HmisSchema::Event.page_type, null: false
+    field :services, HmisSchema::Service.page_type, null: false
+    field :household, HmisSchema::Household, null: true
```

Comment:
@eanders is there a scenario where household is actually null on Enrollment, or are we just mirroring the db schema here? It's non-nullable in the HUD spec.

Reply:
We definitely receive null HouseholdID from other vendors.  We should never generate that, but I _think_ we need to support it without throwing an error if we are trying to display existing data.

Reply:
Hmm okay. Thinking about the interface, if we want to support "add a household member" to an Enrollment that doesn't even have a Household ID yet, we would need to build a special case for that in the UI. (And would you need to manually add the Client who's Enrollment you're already looking at? etc seems like something we'd rather avoid..).

Can we get around this by fixing up data during the migration to the HMIS tables (if Enrollment has no household record, create Household with 1 client)?

Also related:  if you didn't already, can you confirm the nullability of the rest of the fields in this PR on Enrollment/Assessment/Service/Event? I already confirmed that they match HUD requirements, but if any of these have nulls in practice, we should change them here. We generate Typescript types based off this, so the frontend will expect a value and then crash at runtime when it doesn't get one (we can make that crash somewhat graceful and localized on the page, but still).

Reply:
Any enrollment that doesn't have a HouseholdD should get one immediately.  On the HMIS/warehouse side, a household isn't a group of people, just an ID, so yeah, we should be adding them on import (pretty sure we already do.)

You can definitely have an enrollment with no services, events, or assessments.  Not sure I fully understand the null: false in this context.

Reply:
`null: false` just means that we'll probably return an empty array rather than an actual `null` value. That's often cleaner to deal with and more representative of the structure, given that those are has-many fields.

Reply:
Also one thing we could do is assign a household ID at query time if a given enrollment doesn't have one

Reply:
`household` is not a has-many field, though. For non-array fields, I take `null: false` to mean that the api will never return `null` for that field. Based on @eanders most recent response saying that the warehouse does/will create a household ID on import, I think the frontend can always expect a Household to be present. So, I think this should be:

```suggestion
    field :household, HmisSchema::Household, null: false
```

Reply:
Right, that is the case for the singular fields. I was referring to assessments/events/services above, not household.

And I'll go ahead and make this change

Reply:
empty arrays are good

Reply:
Fixed